Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/math-functions/sqrt.adoc

= SQRT
:description: The SQRT() function returns the square root of a given positive number.
:page-topic-type: reference

The `SQRT()` function returns the square root of a given positive number.

== Syntax

The syntax for the `SQRT()` function in Redpanda SQL is:

[source,sql]
----
SQRT(x)
----

The `SQRT()` function requires one argument:

* `x`: A positive number or an expression that evaluates to a positive number.

== Examples

=== SQRT() a positive value

The following example demonstrates how the `SQRT()` function can be used to find the square root of a positive integer:

[source,sql]
----
SELECT SQRT(81);
----

The following result is returned:

[source,sql]
----
+-----+
| f   |
+-----+
| 9   |
+-----+
----

=== SQRT() with an expression

> The following example uses the `SQRT()` function to find the square root of the result of an expression.

Reviewer: ```suggestion
This example shows how to use the `SQRT()` function to find the square root of the result of an expression:
```
